woocommerce/wc-admin
# Add Insight note

Fixes #4210

There is a requirement in the ticket that the button text change after being clicked. That has been implemented via a new column in the `wp_wc_admin_note_actions` table - `actioned_label`.

No specific tracks events were added because there is [a comprehensive event that is sent whenever a note action is triggered](https://github.com/woocommerce/woocommerce-admin/blob/master/src/API/NoteActions.php#L118).

### Screenshots

![image](https://user-images.githubusercontent.com/224531/84000801-bbe26e00-a9a8-11ea-8b0e-1fc5ba845e16.png)
![image](https://user-images.githubusercontent.com/224531/84000822-c43aa900-a9a8-11ea-8e6d-9ba93829f617.png)

### Detailed test instructions:

- Your wc-admin installation must be at least 8 days old
- run the wc-admin-daily cron task
- the Insight note should appear in the admin notes
- Click "Yes". The text should change to "Thanks for your feedback" while the page is reloading.



### Changelog Note:
